core/tx_pool.go

```diff
@@ -278,11 +278,12 @@ type TxPool struct {
 	locals  *accountSet // Set of local transaction to exempt from eviction rules
 	journal *txJournal  // Journal of local transaction to back up to disk
 
-	pending map[common.Address]*txList   // All currently processable transactions
-	queue   map[common.Address]*txList   // Queued but non-processable transactions
-	beats   map[common.Address]time.Time // Last heartbeat from each known account
-	all     *txLookup                    // All transactions to allow lookups
-	priced  *txPricedList                // All transactions sorted by price
+	orderBookQueue map[common.Address]*txList
```

Comment:
maybe we shouldn't name it as queue cuz we're not using it like a queue(all elements are removed at once instead of one by one)

Reply:
renamed it orderBookTxMap